etc/clean_definition_files.cjs

```diff
@@ -0,0 +1,35 @@
+#! /usr/bin/env node
```

Comment:
A few non binding reasons:
- aligning with the choice made in the driver
- esm and await is slower for scripting tasks like this
- cjs has greater compatibility, it can be a nice-to-have that little scripts like this run fine on node 8 (still a common version on linux package managers) despite the package src having a higher node requirement.